frontend/src/scenes/session-recordings/playlist/sessionRecordingsPlaylistLogic.ts

```diff
@@ -596,6 +611,17 @@ export const sessionRecordingsPlaylistLogic = kea<sessionRecordingsPlaylistLogic
                 actions.maybeLoadSessionRecordings('older')
             }
         },
+
+        loadSessionRecordingsFailure: ({ error, errorObject }) => {
+            if (errorObject?.data?.type === 'validation_error' && errorObject?.data?.code === 'type_mismatch') {
+                lemonToast.error(
+                    'One or more property values in your filters are in an incorrect format.' +
+                        ' Please check and adjust them to match the required data types.'
```

Comment:
the perfect thing here would be for the toast to link to the data management properties page... iirc we can set a button on the toast so the person has an onward journey

(even better if it's to the specific property)